iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/datamodel/ScheduleEntity.java

```diff
@@ -15,6 +15,75 @@
 @Table(name = "schedule")
 public class ScheduleEntity {
 
+    public static class ScheduleEntityBuilder {
+
+        private int indexNumber;
+        private int day;
+        private int timeTypeOn;
+        private int timeTypeOff;
+        private LocalTime timeOn;
+        private LocalTime timeOff;
+        private int burningMinutes;
+        private boolean enabled;
+        private RelayEntity relayEntity;
+
+        public ScheduleEntityBuilder(int scheduleNr, RelayEntity relayEntity) {
+            this.indexNumber = scheduleNr;
+            this.relayEntity = relayEntity;
+        }
+
+        public void setIndexNumber(int indexNumber) {
+            this.indexNumber = indexNumber;
+        }
+
+        public void setDay(int day) {
```

Comment:
- A builder should use either "withDay" or just "day" as method name.
- The method should return the builder, so you can chain calls to the builder.

Reply:
Done, corrected the entire builder.